src/app/(proper_react)/Dashboard.stories.tsx

```diff
@@ -27,7 +28,7 @@ const ScanResultMockItem: ScanResult = {
   addresses: [
     {
       city: "123",
-      state: "State",
+      state: "State" as StateAbbr,
```

Comment:
I made a few small corrections to existing mocks - I wasn't sure if you wanted to use the new API mock class here or not @codemist? I can add that if so